wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Content/Text/ConversationTextMessageCell.swift

```diff
@@ -161,7 +177,15 @@ final class ConversationTextMessageCell: UIView, ConversationMessageCell, TextVi
 
         container?.isBubble = isChatBubbleSimpleEnabled
         updateContainerStyle()
-        configureTextColor(forOwnMessage: message?.isSentBySelfUser ?? false)
+        addAccentColorChangeObserver(userSession: object.userSession)
+    }
+
+    private func addAccentColorChangeObserver(userSession: UserSession?) {
+        guard accentColorChangeHandler == nil, let userSession else { return }
+        accentColorChangeHandler = AccentColorChangeHandler
+            .addObserver(userSession: userSession) { [unowned self] _ in
```

Comment:
Would `weak message` work as well? We had some crashes in the past where notification observers were called just a little bit after the UI has been torn down. I'm afraid having an unowned reference to a view might lead to the same issue.
```suggestion
            .addObserver(userSession: userSession) { [weak message] _ in
```

Reply:
Thank you for the review, will do the changes as suggested